tests/models/store.py

```diff
@@ -277,5 +292,158 @@ def test_update_upload_old_revision_new_unit(en_tutorial_po):
     file_name = "tests/data/po/tutorial/en/tutorial_update_old_unit.po"
     _update_from_upload_file(en_tutorial_po, file_name)
 
-    # the new unit has not been added
-    assert len(en_tutorial_po.units) == 1
+    # OLD BEHAVIOUR: the new unit has not been added
```

Comment:
How relevant is it to mention a behavior change here? This would be for docs/release notes, not sure how it fits here though.